tools/create-openbsd-gce-ci.sh

```diff
@@ -40,7 +40,7 @@ mkdir /syzkaller
 echo '/dev/sd1a /syzkaller ffs rw,noauto 1 0' >> /etc/fstab
 
 rm -rf /usr/{src,ports}
-ln -s /syzkaller/src /usr/src
+ln -s /syzkaller/managers/main/kernel /usr/src
```

Comment:
Why are doing this? I would suggest dropping this.

Reply:
> > Sep 27 10:18:38 ci-openbsd vmd[16182]: /etc/vm.conf:6: vm "syzkaller" failed: Invalid argument

> When encountering such a brief error message, check /var/log/daemon for clues or 

The log entry above was not sufficient by itself.

> run vmd in the foreground: # vmd -d

Thanks, this is helpful!
```
$ doas vmd -d
startup
no kernel or disk/cdrom specified
/etc/vm.conf:7: vm "syzkaller" failed: Invalid argument
priv exiting, pid 5319
vmm exiting, pid 6052
control exiting, pid 37334
```

I wonder if a implementing the disk-less feature or documenting /dev/null workaround is in order?